### ISSUE TITLE
Factor binding host fibrinogen

### ISSUE THREAD

**Issue by genegodbold** [jhuapl-bio/pathogenesis-gene-ontology#81]
Please enter your suggested changes or comments here.  If your comment relates to a specific term, please provide the PathGO id or rdfs:label of the term in addition to your comment.

For new term requests, please provide the following information:
## Preferred term label
Factor binding host fibrinogen

## Synonyms
None

## Textual definition
Parasite protein that binds host fibrinogen, a glycoprotein complex that circulates in the blood and which is critical for clotting when acted upon by thrombin to convert it to fibrin.

[There are at least 20 of these for bacteria. Sometimes the bound host fibrinogen allows the inactivation of host complement (indirect), but this is not always the case.]

## Suggested parent term
Along with the newly proposed term "factor binding host plasminogen", this term should be placed under, uh, maybe	PATHGO_0000115 (cellular metabolism disruption factor)?

## Attribution
https://orcid.org/0000-0002-5702-4690

**Comment by genegodbold** [comment]
I have realized in the past few days that these things often, but not always have something to do with bacterial dissemination and is not unrelated to host innate defenses. Clotting and neutrophil extracellular traps (NETs) can be ways that the mammalian host tries to control the location of the parasite to keep it localized, but the parasite develops means to cut the clots or the web of DNA or avoid killing (by the associated cationic histones) with different molecular approaches. These either free the bacteria, allowing them to disseminate, or they keep it alive.

**Comment by jproesch** [comment]
@genegodbold there is a Go term that covers fibrinogen binding, not sure we need to recreate that in PathGO, although the consequence of that binding is likely fair game.  Is it typically activation? Or does the binding sequester fibrinogen?

**Comment by genegodbold** [comment]
Hey @jproesch 
Sounds good--though it might be nice to specify binding fibrinogen **in a host**. Most of the fibrinogen binding proteins, and there are over a dozen for bacteria, seem to mostly be adhesins.

**Comment by jproesch** [comment]
@genegodbold true.  This is something we are trying to figure out where to draw the line on in terms of overlap with GO.   I'm going to hold this Issue for now, we can revisit.

**Comment by genegodbold** [comment]
@jproesch If one is trying to capture activity of a parasite or an external actor, you need a term that is uncontaminated with normal homeostatic mechanisms.
